src/sass/_clamp.scss

```diff
@@ -0,0 +1,19 @@
+.clamp {
+  display: -webkit-box;
```

Comment:
Doesn't this mean only webkit browser would support this functionality?

Reply:
https://caniuse.com/#feat=css-line-clamp

Reply:
> As there is no specification and the property is dependent on an outdated implementation of flexbox (hence display: -webkit-box), an alternative solution may at some point replace it.

Reply:
Not a big fan of that.